tests/baselines/reference/api/typescript.d.ts

```diff
@@ -123,6 +123,7 @@ declare namespace ts {
                 ProvideInlayHints = "provideInlayHints",
                 WatchChange = "watchChange",
                 MapCode = "mapCode",
+                /** @deprecated */
```

Comment:
Maybe just make these internal in the API? This was never in a stable release so nobody should depend on compiling it.

Reply:
Even deleting the types below may be okay given they aren't used.